services/app-api/src/resolvers/questionAnswers/createQuestion.test.ts

```diff
@@ -0,0 +1,303 @@
+                    ]),
+                }),
+                DMCPQuestions: expect.objectContaining({
+                    totalCount: 0,
+                    edges: [],
+                }),
+                OACTQuestions: expect.objectContaining({
+                    totalCount: 0,
+                    edges: [],
+                }),
+            })
+        )
+    })
+    it('returns an error package status is DRAFT', async () => {
+        const stateServer = await constructTestPostgresServer()
+        const cmsServer = await constructTestPostgresServer({
+            context: {
+                user: testUserCMS,
+            },
+        })
+
+        const newSubInput: CreateHealthPlanPackageInput = {
+            programIDs: [
+                '5c10fe9f-bec9-416f-a20c-718b152ad633',
+                '037af66b-81eb-4472-8b80-01edf17d12d9',
+            ],
+            riskBasedContract: false,
+            submissionType: 'CONTRACT_ONLY',
+            submissionDescription: 'A real submission',
+            contractType: 'BASE',
+        }
+
+        const draftPkg = await stateServer.executeOperation({
+            query: CREATE_HEALTH_PLAN_PACKAGE,
+            variables: {
+                input: newSubInput,
+            },
+        })
+
+        const pkg = draftPkg.data?.createHealthPlanPackage.pkg
+
+        const input = {
+            pkgID: pkg.id,
+            documents: [
+                {
+                    name: 'Test Question',
+                    s3URL: 'testS3Url',
+                },
+            ],
+        }
+
+        const createdQuestion = await cmsServer.executeOperation({
+            query: CREATE_QUESTION,
+            variables: { input },
+        })
+
+        expect(createdQuestion.errors).toBeDefined()
+        expect(
+            createdQuestion.errors && createdQuestion.errors[0].message
+        ).toBe(
+            'Issue creating question for health plan package. Message: Cannot create question for health plan package in DRAFT status'
+        )
+    })
+    it('returns an error if a state user attempts to create a question for a package', async () => {
+        const stateServer = await constructTestPostgresServer()
+
+        const submittedPkg = await createAndSubmitTestHealthPlanPackage(
+            stateServer
+        )
+
+        const input = {
+            pkgID: submittedPkg.id,
+            documents: [
+                {
+                    name: 'Test Question',
+                    s3URL: 'testS3Url',
+                },
+            ],
+        }
+
+        const createdQuestion = await stateServer.executeOperation({
+            query: CREATE_QUESTION,
+            variables: { input },
+        })
+
+        expect(createdQuestion.errors).toBeDefined()
+        expect(
+            createdQuestion.errors && createdQuestion.errors[0].message
+        ).toBe('user not authorized to create a question')
+    })
+    it('returns error on invalid package id', async () => {
```

Comment:
Since you added the NOT_FOUND extension, let’s have the test that checks that comes through